main.go

```diff
@@ -95,7 +88,7 @@ func main() {
 		LeaderElectionID:       "c1f6bfd2.kueue.x-k8s.io",
 	}
 	var err error
-	config := defaultConfiguration()
+	config := configv1alpha1.DefaultConfiguration()
 	if configFile != "" {
 		options, err = options.AndFrom(ctrl.ConfigFile().AtPath(configFile).OfKind(&config))
```

Comment:
Does this actually preserve the values that were already set?

Maybe we should have a function `SetDefaults` that applies after the file was loaded.

And unit test that.

Reply:
I would prefer we do the same we do in upstream, which sets defaults after loading the file.

Reply:
Could you please provide some examples in upstream? In kube-scheduler, we also call `Default()` first then we complete the other configurations.

Reply:
Can you add unit tests?